x-pack/plugins/triggers_actions_ui/public/application/lib/value_validators.ts

```diff
@@ -43,3 +44,66 @@ export const isValidUrl = (urlString: string, protocol?: string) => {
     return false;
   }
 };
+
+export function getConnectorWithInvalidatedFields(
+  connector: UserConfiguredActionConnector<Record<string, unknown>, Record<string, unknown>>,
+  configErrors: IErrorObject,
+  secretsErrors: IErrorObject,
+  baseConnectorErrors: IErrorObject
+) {
```

Comment:
If I create an email connector and enter an invalid Sender field, I get immediate feedback on the form that "Sender is not a valid email address." but then if I press Save, my (invalid) entry is cleared and the error just says "Sender is required.". Would it be better to keep the user input so the user can fix it?

Reply:
Great catch! I will fix it.